app/uk/gov/hmrc/helptosavefrontend/util/package.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend
+
+import cats.data.EitherT
+import play.api.libs.json.{JsError, Reads}
+import play.mvc.Http
+import uk.gov.hmrc.play.http.HeaderCarrier
+import uk.gov.hmrc.helptosavefrontend.util.JsErrorOps._
+import scala.concurrent.{ExecutionContext, Future}
+
+package object util {
+
+  type NINO = String
```

Comment:
Having it as a type avoids having to read and write ninos like
```
val nino = Nino("ABC123")
...
val nino2 = nino.id
```
Also for JSON reading and writing having this:
```
case class Person(name: String, nino: NINO)
```
would result in the default `Reads` and `Writes` instances expecting and writing this kind of JSON:
```
{
  "name" : "Andy"
  "nino" : {
     "id": "ABC123"
  }
}
```
instead of
```
{
  "name" : "Andy"
  "nino" : "ABC123"
}
```
Although we do get type safety with having it as a case class.